content/03.methods.md

```diff
@@ -108,6 +109,14 @@ For input intervals to Lancet, a reference BED was created by using only the UTR
 These intervals were then padded by 300 bp on each side during Lancet variant calling.
 Per recommendations by the New York Genome Center [@doi:10.1101/623702], for WGS samples, the Lancet input intervals described above were augmented with PASS variant calls from Strelka2 and Mutect2 as validation.
 
+##### TCGA data
+
+We used three variant callers to call SNVs and INDELS in TCGA data from WXS data: Strelka2, Mutect2  and Lancet. Not  all BAM files within the same disease group were  captured  with the same WXS kit. We retreived individual target  capture kit BED files for each BAM file from 
+[`GDC portal`](https://api.gdc.cancer.gov/files). The input  BED files were  padded by 100 bp on each side during Strelka2 and Mutect2 variant calling and 400bp for Lancet variant calling to accomodate local reassembly.
+The final Strelka2 VCF was filtered for PASS variants.
+Mutect2 from GATK v4.1.1.0 was run following Broad best practices outlined from their Workflow Description Language (WDL) [@url:https://github.com/broadinstitute/gatk/blob/4.1.1.0/scripts/mutect2_wdl/mutect2.wdl].
```

Comment:
This appears to be a direct copy of the section above. What about not breaking out the two sections and then just having one paragraph describing differences, if any, between the two workflows?

Reply:
Hi @cgreene Thank you for your comments! I updated the section where I combined both TCGA and PBTA under SNV calling.

I also wanted to add how many samples we chose from TCGA under each disease group and primary diagnosis. Do you have a recommendation on where that would be most appropriate to add? Is `Data generation` the right spot? 
https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/03.methods.md#data-generation